src/Sylius/Bundle/CoreBundle/Resources/config/app/sylius.yml

```diff
@@ -679,3 +683,18 @@ sylius_rbac:
         administrator: [catalog_manager, sales_manager, marketing_manager, accounts_manager, shipping_manager, content_editor, tax_manager, it, support]
         it: [security_auditor, developer]
 
+sylius_user:
+    resetting:
+        token:
+            ttl: PT1S
```

Comment:
I remember we set this just for testing, should be more sensible default or even configurable in parameters.yml.